e2e/menu_items_inline/information_callouts.spec.js

```diff
@@ -10,9 +10,8 @@ test.fixme(
   async ({ page }) => {
     await page.getByText("^", { exact: true }).click();
     await expect(page.locator(".menubar")).toBeVisible();
-    const enabledMenuButtons = ["p", "H2", "H3", "$A", "$CTA", "$C", "$E", "%"];
-    const disabledMenuButtons = ["“”", "^", "1.", "-"];
-
+    const enabledMenuButtons = ["p", "H2", "H3", "$A", "$CTA", "$E", "%"];
+    const disabledMenuButtons = ["“”", "^", "1.", "-", "$C"];
```

Comment:
Not clear to me why this test or the warning callout test have changed?

Reply:
It's because information and warning callouts can no longer be wrapped by contact blocks